src/breadcrumbs/index.jsx

```diff
@@ -1,25 +1,30 @@
 import React from 'react';
+import { omit } from 'lodash';
 import Link from '../link';
 import Snippet from '../snippet';
 
-export const Breadcrumb = ({
-  crumb = {},
-  link = false
-}) => {
-  return <li className="govuk-breadcrumbs__list-item">
-  {
-    link ?
-      <Link page={crumb} label={<Snippet>{`breadcrumbs.${crumb}`}</Snippet>} /> :
-      <Snippet>{`breadcrumbs.${crumb}`}</Snippet>
+export const Breadcrumb = ({ crumb = {}, link = false }) => {
+  let props = null;
+
+  if (typeof crumb === 'object' && crumb !== null) {
+    props = omit(crumb, 'label');
+    crumb = crumb.label;
```

Comment:
had to add this code so we can pass through the project here: https://github.com/UKHomeOffice/asl-pages/pull/378/files#diff-203c058d076e3990780ee39e62db89dfR14

because the project title is used in the breadcrumb (https://github.com/UKHomeOffice/asl-pages/pull/378/files#diff-208db7cd30d5553da5170774af9321e7R46).

We can't just rely on the `model` prop because sometimes the project is the model and sometimes the version is the model.

Reply:
seems legit